packages/ApiTests/src/viewers/filters.ts

```diff
@@ -0,0 +1,74 @@
+import * as ui from 'datagrok-api/ui';
+import * as grok from 'datagrok-api/grok';
+import * as DG from 'datagrok-api/dg';
+
+import {
+  after,
+  before,
+  category,
+  expect,
+  test,
+  delay,
+  awaitCheck,
+  expectArray
+} from '@datagrok-libraries/utils/src/test';
+import {TableView} from 'datagrok-api/dg';
+import {_package} from '../package-test';
+
+import $ from 'cash-dom';
+
+category('Viewers: Filters', () => {
+  const csv1: string = `id1,id2,id3
+id1_0001,id2_001,id3_1
+id1_0002,id2_002,id3_2
+id1_0003,id2_003,id3_3`;
+
+  test('twoCategorical', async () => {
+    const df: DG.DataFrame = DG.DataFrame.fromCsv(csv1);
+    const view: TableView = grok.shell.addTableView(df);
+
+    const filterList: { [p: string]: string }[] = [
+      {column: 'id1', type: DG.FILTER_TYPE.CATEGORICAL, label: 'id1 label'},
+      {column: 'id2', type: DG.FILTER_TYPE.CATEGORICAL, label: 'id2 label'},
+    ];
+    const filtersViewer = view.filters({filters: filterList}) as DG.Viewer;
+    const filtersViewerDn: DG.DockNode = view.dockManager.dock(
```

Comment:
what is the filtersViewerDn variable for?

Reply:
removed